packages/nuxt/package.json

```diff
@@ -30,20 +30,20 @@
     "prepack": "pnpm build"
   },
   "devDependencies": {
-    "@nuxt/module-builder": "^0.8.0",
-    "@nuxt/test-utils": "^3.12.1",
+    "@nuxt/module-builder": "^0.8.4",
+    "@nuxt/test-utils": "^3.15.1",
     "@silverhand/eslint-config": "^6.0.1",
-    "@vitest/coverage-v8": "^1.6.0",
+    "@vitest/coverage-v8": "^2.1.8",
     "@vue/test-utils": "^2.4.4",
     "eslint": "^8.57.0",
-    "h3": "^1.11.1",
+    "h3": "^1.13.0",
     "happy-dom": "^15.10.2",
     "lint-staged": "^15.0.0",
-    "nuxt": "^3.12.4",
+    "nuxt": "^3.15.0",
     "prettier": "^3.0.0",
     "typescript": "^5.3.3",
-    "vitest": "^1.6.0",
-    "vue": "^3.4.19"
+    "vitest": "^2.1.8",
+    "vue": "^3.5.13"
```

Comment:
Sync vue and vitest version in other packages

Reply:
We can do it later, this PR is for nuxt.